<launch>

    <!--    <machine name="local_alt" address="localhost" default="true"/>-->
    <param name="Voc_file" type="string" value="/home/daybeha/Documents/github/ORB_SLAM3_detailed_comments-master/Vocabulary/ORBvoc.txt"/>
    <param name="config_file" type="string" value="$(find ORB_SLAM3)/config/KITTI00-02.yaml"/>
<!--    <param name="config_file" type="string" value="$(find ORB_SLAM3)/config/KITTI04-12.yaml"/>-->

    <param name="do_rectify" type="bool" value="false"/>
    <param name="img_l" type="string" value="/kitti/camera_gray_left/image_raw"/>
    <param name="img_r" type="string" value="/kitti/camera_gray_right/image_raw"/>


    <param name="dataset" type="string" value="urban"/>
    <!--    traj_format 0:kitti 1:tum-->
    <param name="traj_format" type="int" value="0"/>
    <param name="traj_outpath" type="string" value="$(find ORB_SLAM3)/results/"/>


    <node name="Stereo_pub" pkg="ORB_SLAM3" type="Stereo_pub" output="screen" cwd="node" required="true"/>

    <!-- 启动rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ORB_SLAM3)/config/rviz.rviz" />

<!--    <node pkg="rosbag" type="record" name="record"-->
<!--          args="/orb/path /orb/odom /camera/image_raw/compressed -O stereo_kitti.bag"/>-->
</launch>